packages/@react-spectrum/menu/src/ContextualHelpTrigger.tsx

```diff
@@ -151,7 +151,7 @@ function ContextualHelpTrigger(props: InternalMenuDialogTriggerProps): ReactElem
         containerPadding={0}
         hideArrow
         enableBothDismissButtons>
-        <FocusScope restoreFocus>
+        <FocusScope>
```

Comment:
Due to the above FocusScope change, we don't want to restoreFocus from the unavailable menu dialog because that will move focus back to the parent menu item when tabbing out. There is a test that asserts that tabbing out moves to the item after the root menu trigger. Perhaps we might want to change this behavior at some point. For now, just retaining existing behavior.